assets/scss/_cookie-banner.scss

```diff
@@ -17,10 +17,7 @@ $cookie-banner-bg: #bff3ff;
 	letter-spacing: 0.02em;
 	font-size: 13px;
 	font-family: $font-primary;
-	padding-top: 35px;
-	padding-left: 25px;
-	padding-right: 25px;
-	padding-bottom: 25px;
+	padding: 35px 25px 25px 25px;
```

Comment:
Thank you for your interest and contribution.
Yes we can refactor this, but IMO the as-is serves the same functionality, and explicit and clear so it would help people understand the meaning of each field and corresponding value more easily.

Reply:
Okay @jihoon-seo,
Thankyou for reviewing.